pkg/procedure/procedure.go

```diff
@@ -22,19 +22,20 @@
 package procedure
 
 import (
-	"fmt"
 	"strings"
 )
 
+const _separator = "::"
+
 // ToName gets the procedure name we should use for a method
 // with the given service name and method name.
 func ToName(serviceName string, methodName string) string {
-	return fmt.Sprintf("%s::%s", serviceName, methodName)
+	return serviceName + _separator + methodName
```

Comment:
In general, isn't this fully pre-computable and cacheable at startup? All yarpc configuration is static, right?

Reply:
I agree that by pre-computing this information early and storing it in the client object, we might further reduce allocations. I'll investigate this possibility in a separate PR.